src/pages/apply.js

```diff
@@ -0,0 +1,1169 @@
+              <RequiredError />
+            )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <Checkbox>
+            <CheckboxInput
+              type="checkbox"
+              name="POCisAuthorisedSignatory"
+              defaultChecked={formState.POCisAuthorisedSignatory.value}
+              onChange={handleCheckBoxChange}
+              required
+            />
+            Is the point of contact also the authorised signatory?
+          </Checkbox>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Name, job title, and email address of the authorised signatory{" "}
+            <Required>*</Required>
+          </span>
+          <div>
+            <small>
+              (e.g. John Smith, CEO, john@mycompany.com. This is the person who
+              will sign the contract. They must be someone who can sign
+              contracts on behalf of the entity.)
+            </small>
+          </div>
+          <Input
+            type="text"
+            name="authorisedSignatoryInformation"
+            value={formState.authorisedSignatoryInformation.value}
+            onChange={handleInputChange}
+            maxLength="255"
+            onBlur={handleTouched}
+            required
+          />
+          <ErrorDiv>
+            {formState.authorisedSignatoryInformation.isTouched &&
+              !formState.authorisedSignatoryInformation.isValid && (
+                <RequiredError />
+              )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Email <Required>*</Required>
+          </span>
+          <Input
+            type="email"
+            name="contactEmail"
+            value={formState.contactEmail.value}
+            onChange={handleInputChange}
+            maxLength="255"
+            onBlur={handleTouched}
+            required
+          />
+          <ErrorDiv>
+            {formState.contactEmail.isTouched &&
+              !formState.contactEmail.isValid && <EmailError />}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Phone number <Required>*</Required>
+          </span>
+          <div>
+            <small>Phone number with country code (format - eg.: +1)</small>
+          </div>
+          <Input
+            type="text"
+            name="phone"
+            value={formState.phone.value}
+            onChange={handleInputChange}
+            maxLength="255"
+            onBlur={handleTouched}
+            required
+          />
+          <ErrorDiv>
+            {formState.phone.isTouched && !formState.phone.isValid && (
+              <RequiredError />
+            )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Which gender pronoun do you use? <Required>*</Required>
+          </span>
+          <StyledSelect
```

Comment:
This field should probably be just a dropdown select vs. having search functionality.